airflow/www/static/js/dag/details/index.tsx

```diff
@@ -134,6 +153,12 @@ const Details = ({ openGroupIds, onToggleGroups, hoveredTaskState }: Props) => {
     [setSearchParams, searchParams, showLogs, showMappedTasks, taskId]
   );
 
+  useEffect(() => {
+    if ((!taskId || isGroup) && tabIndex > 3) {
+      onChangeTab(1);
+    }
+  }, [runId, taskId, tabIndex, isGroup, onChangeTab]);
```

Comment:
Is `runId` necessary here ?

Reply:
Updated. There was an existing bug with resetting the tabs.